pkg/types/types.go

```diff
@@ -74,4 +74,6 @@ const (
 	RefVCS          RefType = "vcs"
 	RefIssueTracker RefType = "issue-tracker"
 	RefOther        RefType = "other"
+
+	NonSeparableLicensePrefix = "non-separable: "
```

Comment:
We already have [file://](https://github.com/aquasecurity/go-dep-parser/blob/6eff4752bcbe21782f5521ad234b0ea501ad8410/pkg/python/packaging/parse.go#L47) prefix.

Perhaps we want to create new struct for [License](https://github.com/aquasecurity/go-dep-parser/blob/6eff4752bcbe21782f5521ad234b0ea501ad8410/pkg/types/types.go#L13) field.

just as idea:
``` go
type License struct {
	Value string
	Type  licenseType
}

type licenseType string

const (
	fileLicenseType             licenseType = "file" // filename for license file
	textLicenseType             licenseType = "text" // text of license (for split, classifier, etc.)
	nonSeparableTextLicenseType licenseType = "nonSeparable-text" // text of license without possible to split
	formattedLicenseType        licenseType = "formattedLicenses" // formatted licenses (e.g. in SPDX format)
)
```

Reply:
It sounds reasonable. We can go with that approach.

Reply:
@knqyf263 I have added `licenseType` to this PR.
I also updated https://github.com/aquasecurity/trivy/pull/5211 to show the required changes for Trivy.
#5211 requires some refactoring. I will do this if you confirm that we are ready for these changes.